src/ray/common/id.cc

```diff
@@ -34,14 +47,47 @@ WorkerID ComputeDriverIdFromJob(const JobID &job_id) {
       std::string(reinterpret_cast<const char *>(data.data()), data.size()));
 }
 
+ObjectID ObjectID::FromPlasmaIdBinary(const std::string &from) {
+  constexpr size_t plasma_id_length = kUniqueIDSize;
```

Comment:
where is this `kUniqueIDSize` defined?

Reply:
constants.h

Reply:
oh, I misread it. I thought you deleted this var. Should we remove it to `kPlasmaIdSize`? Because there's no `unique id` in ray right now.

Reply:
There still have unique id on the codebase now. Whatever `kPlasmaIdSize` seems better.